### PR TITLE
Add privacy policy within repo

### PR DESCRIPTION
**Pull request by rougetimelord**
Yippie I get to use my rules lawyering brain for good instead of scaring my friends :))))

### CODE REVIEW

File: privacy.md
Comment (cooljeanius) [inline privacy.md:3]:
```suggestion
This privacy policy ("policy") will help you understand how Blue Blocker collects, uses, and protects the data you provide when you use the Blue Blocker extension/addon (herein: "the Extension").
```

File: privacy.md
Comment (cooljeanius) [inline privacy.md:27]:
```suggestion
We are committed to safeguarding all data that you provide to the Extension. We have done everything in our power to prevent data theft, unauthorized access, and disclosure by storing as much data locally as is feasible, and only using well understood browser APIs to store your data.
```

**Review by cooljeanius — CHANGES_REQUESTED** [review]
just 2 minor typoes noticed

**Review by cooljeanius — APPROVED** [review]
LGTM, assuming that it will be kept in sync with what's submitted to Firefox

**Comment by kheina** [comment]
hold off until I can figure something out with the release workflow

**Comment by cooljeanius** [comment]
> hold off until I can figure something out with the release workflow

Any idea of a timeframe for this?

**Pull request merged**
